modules/installation-launching-installer.adoc

```diff
@@ -534,6 +534,37 @@ INFO Time elapsed: 36m22s
 * It is recommended that you use Ignition config files within 12 hours after they are generated because the 24-hour certificate rotates from 16 to 22 hours after the cluster is installed. By using the Ignition config files within 12 hours, you can avoid installation failure if the certificate update runs during installation.
 ====
 
+ifdef::azure-private[]
+* If you are deploying a private image registry with your private Microsoft Azure cluster, you can enter the following command to check if a `privateEndpointName` resource has been created for the image registry:
++
+[source,terminal]
+----
+$ oc get config.image/cluster -o yaml
+----
++
+.Example output
++
+[source,yaml]
+----
+# ...
+  storage:
+    azure:
+      accountName: <account_name>
```

Comment:
this parameter is optional, if not provided the operator will generate one.

Reply:
nevermind my comment, I missed the fact that this was an output

Reply:
Addressed